### PR TITLE
WinUI-2902 : Changed the content and images in rating and busyindicator control.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:

- Changed the content in the precision MD file in Rating control.
- Changed the image in the overview in BusyIndicator control.

### CODE REVIEW

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Rating/Precision.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The rating value can be provided with three options such as full, half, and exac
 
 ## Full
 
-Rating value can be provided with whole precision.
+Rating value can be provided with nearest rounded value.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:16]: Rating value can be provided with the nearest rounded value.

File: winui/Rating/Precision.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ rating.ItemsCount = 5;
 
 ## Half
 
-Rating value can be provided with half precision.
+Rating value can be provided with nearest half value.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:40]: The rating value can be provided with the nearest half value.

File: winui/Rating/Precision.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ rating.ItemsCount = 5;
 
 ## Exact
 
-Rating value can be provided with exact precision.
+Rating value can be provided with exact value.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:64]: The rating value can be provided with the exact value.

File: winui/Rating/Precision.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:16]: Changed the content as mentioned above.

File: winui/Rating/Precision.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:40]: Changed the content as mentioned above.

File: winui/Rating/Precision.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:64]: Changed the content as mentioned above.

**Pull request merged**